docs/t-sql/language-elements/declare-cursor-transact-sql.md

```diff
@@ -151,12 +151,15 @@ Specifies that positioned updates or deletes made through the cursor are guarant
 
 #### OPTIMISTIC
 
-Specifies that positioned updates or deletes made through the cursor don't succeed, if the row was updated since it was read into the cursor. [!INCLUDE [ssNoVersion](../../includes/ssnoversion-md.md)] doesn't lock rows as they are read into the cursor. It instead uses comparisons of **timestamp** column values, or a checksum value if the table has no **timestamp** column, to determine whether the row was modified after it was read into the cursor. If the row was modified, the attempted positioned update or delete fails. `OPTIMISTIC` can't be specified if `FAST_FORWARD` is also specified.
+Specifies that positioned updates or deletes made through the cursor don't succeed, if the row was updated since it was read into the cursor. [!INCLUDE [ssNoVersion](../../includes/ssnoversion-md.md)] doesn't lock rows as they are read into the cursor. It instead uses comparisons of **timestamp** column values, or a checksum value if the table has no **timestamp** column, to determine whether the row was modified after it was read into the cursor. If the row was modified, the attempted positioned update or delete fails. `OPTIMISTIC` can't be specified if `FAST_FORWARD` is also specified, and is ignored if `STATIC` is also specified.
```

Comment:
I would suggest something like:
If the row was modified, the attempted positioned update or delete fails. `OPTIMISTIC` can't be specified if `FAST_FORWARD` is also specified.
 
If `STATIC` is specified, the `OPTIMISTIC` cursor property will be and is ignored if `STATIC` is also specified and the cursor will be converted to the equivalent of the combination of using the `STATIC` and `READ_ONLY` properties or `STATIC` and `FORWARD_ONLY` properties.